src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -335,6 +336,18 @@ const SimOptionsPage = () => {
                         />
                     </div>
 
+                    <div className="py-4 flex flex-row justify-between items-center">
+                        <span>
+                            <span className="text-lg text-gray-300">Enable MCDU Server Connection (deactivates after 5min)</span>
```

Comment:
**thought:** "(deactivates after 5min)" misses the point about that being in case of errors. I would rephrase it to:

"(deactivates after 5 minutes if no successful connection)"